spec/features/applications/index_spec.rb

```diff
@@ -0,0 +1,25 @@
+require "rails_helper"
+
+RSpec.describe "the application index" do
+  before(:each) do
+    @victor_app = Application.create!(applicant_name: "Victor Antonio Sanchez", address: "97 Jaffa Road", city: "Jerusalem", state: "Israel", zip_code: "9103401", description: "Because I'm rich! :)")
+    @shelter_1 = Shelter.create(name: "Aurora shelter", city: "Aurora, CO", foster_program: false, rank: 9)
+    @pet_1 = @shelter_1.pets.create!(name: "Mr. Pirate", breed: "tuxedo shorthair", age: 5, adoptable: true)
+    @pet_2 = @shelter_1.pets.create!(name: "Clawdia", breed: "shorthair", age: 3, adoptable: true)
+    @pet_3 = @shelter_1.pets.create!(name: "Ann", breed: "ragdoll", age: 3, adoptable: false)
+    @victor_app.pets << [@pet_1, @pet_2]
+  end
+  it "display's the applicants info" do
+
+    visit "/applications"
+
+    expect(page).to have_content("Victor Antonio Sanchez")
```

Comment:
This test could be more robust if it included multiple applications, to verify that the index page shows *all* the applications in the db, not just one